openvdb/tools/Activation.h

```diff
@@ -0,0 +1,153 @@
+// Copyright Contributors to the OpenVDB Project
+// SPDX-License-Identifier: MPL-2.0
+//
+/// @file   Activation.h
+///
+/// @brief  Implementation of topological activation/deactivation
+///
+/// @author Ken Museth
+///
```

Comment:
This file is a copy/paste of the activation/deactivation methods out of Morphology.h and can be mostly be ignored